### DISCUSSION TITLE
스터디 참여자 리스트 (1기)

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#52] [공지사항]
1. https://github.com/SamTheKorean
2. https://github.com/leokim0922
3. https://github.com/Invidam
4. https://github.com/dev-jonghoonpark
5. https://github.com/leezer94
6. https://github.com/JinhyukKo
7. https://github.com/claeo001
8. https://github.com/gumbee89
9. https://github.com/bky373
10. https://github.com/BanseokSuh
11. https://github.com/kiwi-choe
12. https://github.com/mand2
13. https://github.com/JoyJaewon
14. https://github.com/wclee2265
15. https://github.com/sounmind
16. https://github.com/nhistory
17. https://github.com/bmh8993
18. https://github.com/yolophg
19. https://github.com/saysimple0828
20. https://github.com/Jaeyeonnn
21. https://github.com/official-Cherry
22. https://github.com/bhyun-kim
23. https://github.com/koreas9408
24. https://github.com/meoooh
25. https://github.com/2duckchun
26. https://github.com/WhiteHyun
27. https://github.com/nakyeonko3

**Comment by DaleSeo** [comment]
스터디 참여를 희망하시는 분들은 자신의 GitHub 프로필 링크를 댓글로 달아주세요 😃

**Comment by bky373** [comment]
안녕하세요 스터디 참여 희망합니다.
- https://github.com/bky373

**Comment by BanseokSuh** [comment]
안녕하세요! 스터디 참여 희망합니다!
- https://github.com/BanseokSuh

**Comment by wonder-choe** [comment]
안녕하세요~ 스터디 참여하고 싶습니다~
https://github.com/kiwi-choe

**Comment by JoyJaewon** [comment]
안녕하세요! 스터디 참여희망합니다~~
https://github.com/JoyJaewon

**Comment by SamTheKorean** [comment]
> 스터디 참여를 희망하시는 분들은 자신의 GitHub 프로필 링크를 댓글로 달아주세요 😃

위 댓글에 추가된 인원들은 제가 엄지척 표시해놓고 readme에도 업데이트 하겠습니다.

**Comment by wclee2265** [comment]
안녕하세요. 스터디 참여 희망합니다.
[https://github.com/wclee2265](https://github.com/wclee2265)

**Comment by sounmind** [comment]
안녕하세요~ 스터디 참여 희망합니다.
- https://github.com/sounmind

**Comment by nhistory** [comment]
안녕하세요! 스터디 참여 희망합니다.
https://github.com/nhistory

**Comment by bmh8993** [comment]
안녕하세요 스터디 참여 희망합니다
https://github.com/bmh8993

**Comment by yolophg** [comment]
안녕하세요, 스터디 참여 희망합니다
https://github.com/yolophg

**Comment by say828** [comment]
안녕하세요 달레님! 스터디 참여 희망합니다!
https://github.com/saysimple0828

**Comment by Jaeyeonnn** [comment]
안녕하세요! 스터디 참여 희망합니다!
https://github.com/Jaeyeonnn

**Comment by official-Cherry** [comment]
안녕하세요 ! 스터디 참여 희망합니다 : D
https://github.com/official-Cherry

**Comment by sbk1014** [comment]
안녕하세요! 스터디 참여 희망합니다!
https://github.com/bhyun-kim

**Comment by yolophg** [comment]
@Bumsu-Yi 
> 19. https://github.com/saysimple0828
> 20. https://github.com/saysimple0828

https://github.com/saysimple0828 님이 중복해서 추가 된 것 같아 코멘트 드립니다!

**Comment by SamTheKorean** [comment]
> @Bumsu-Yi
> 
> > 19. https://github.com/saysimple0828
> > 20. https://github.com/saysimple0828
> 
> https://github.com/saysimple0828 님이 중복해서 추가 된 것 같아 코멘트 드립니다!

반영했습니다! 감사합니다!

**Comment by koreas9408** [comment]
안녕하세요 :) 스터디 참여 희망합니다!
https://github.com/koreas9408

**Comment by meoooh** [comment]
안녕하세요. 스터디 참여 희망합니다.

https://github.com/meoooh

**Comment by 2duckchun** [comment]
안녕하세요! 스터디 참여 희망합니다!

https://github.com/2duckchun

**Comment by WhiteHyun** [comment]
스터디 참여 희망합니다 🫡
https://github.com/WhiteHyun

**Comment by nakyeonko3** [comment]
저도 스터디 참여 희망합니다.
https://github.com/nakyeonko3

**Comment by DaleSeo** [comment]
충분한 인원이 모여서 스터디 멤버 모집은 여기까지 마감을 하고, 지금부터 댓글을 달아주신 분들은 대기자로 추가해드리도록 하겠습니다. 많은 관심과 성원에 감사드립니다! 🙇‍♂️

**Comment by BEMELON** [comment]
대기자 등록합니다~ 참여 희망합니다!
https://github.com/BEMELON

**Comment by lheesung** [comment]
스터디 참여 대기 등록합니다!!
https://github.com/lheesung

**Comment by ltekean** [comment]
안녕하세요 대기자 등록합니다!
https://github.com/ltekean

**Comment by argenturatbek** [comment]
안녕하세요! 스터디 참여 희망합니다!
https://github.com/argenturatbek

**Comment by colorxxx** [comment]
안녕하세요 스터디참여 희망합니다
https://github.com/colorxxx

**Comment by taekwon-dev** [comment]
안녕하세요 대기자 등록합니다!
https://github.com/taekwon-dev

**Comment by chriskr7** [comment]
안녕하세요. 스터디 대기자 등록합니다.

https://github.com/chriskr7

**Comment by zerone-jang** [comment]
안녕하세요!
스터디 대기 신청합니다.

**Comment by 0-Chan** [comment]
스터디 참여 신청해요!
https://github.com/0-Chan

**Comment by YeonguChoe** [comment]
### 참가 신청합니다. 혼자서 Blind75 풀고 있습니다.
[깃 허브](https://github.com/yeonguchoe)

**Comment by Rosa-Kang** [comment]
안녕하세요, 스터디 참여(/대기) 희망합니다.

- https://github.com/Rosa-Kang

**Comment by movie5** [comment]
안녕하세요 스터디 참여 희망합니다!
https://github.com/movie5

**Comment by jdalma** [comment]
안녕하세요 스터디 2기 참여 희망합니다~
[https://github.com/jdalma](https://github.com/jdalma)

**Comment by sunaleed** [comment]
안녕하세요 스터디 참여 희망합니다.
https://github.com/dongsunleej

**Comment by seona926** [comment]
안녕하세요, 스터디 참여 희망합니다!
https://github.com/seona926

**Comment by DaleSeo** [comment]
스터디 2기 모집이 시작되어 대기자는 여기까지 받고 본 디스커션은 닫도록 하겠습니다.

@BEMELON, @lheesung, @ltekean, @argenturatbek, @colorxxx, @taekwon-dev, @chriskr7, @GyeongChan-Jang, @0-Chan, @YeonguChoe, @Rosa-Kang, @movie5, @jdalma, @dongsunleej, @seona926
여전히 스터디 참여에 관심이 있으시다면 아래 디스커션에 Discord 유저네임을 댓글로 달아주시면 감사하겠습니다!

- https://github.com/DaleStudy/leetcode-study/discussions/209

**Comment by hn8423** [comment]
스터디 참여 희망합니다!
https://github.com/hn8423
